src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -40,7 +40,7 @@ public static int Start(RunspaceConfiguration configuration, string bannerText,
 
         /// <summary>
         /// 
-        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace
+        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace (SP!)
```

Comment:
Sorry, I can't reason through that right now. I don't see "MSHManagedEntrace" in any obvious form there.

My question (for the purposes of the task at hand) is simply: is your string "Frozen" such that I should discard any attempts at changing it?